Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -1494,7 +1489,8 @@ public override void ResizeScreen ()
 				// If n is 2, clear entire screen (and moves cursor to upper left on DOS ANSI.SYS).
 				// If n is 3, clear entire screen and delete all lines saved in the scrollback buffer
 				// DO NOT USE 3J - even with the alternate screen buffer, it clears the entire scrollback buffer
-				Console.Out.Write ("\x1b[3J");
+				//Console.Out.Write ("\x1b[1;1H");
+				Console.Out.Write ("\x1b[0J");
```

Comment:
Still don't understand why this one. When the `EnableConsoleScrolling` is enable the `Console.Out.Write ("\x1b[3J")` is the one that work with it by only showing the `Terminal.Gui` on scrolling, instead mixing the app with terminal commands that user has typed before run the app.

Reply:
* `[0J` does not clear the back buffer
* `[3J` DOES clear the back buffer

For the most common use case on Windows, which is Windows Terminal with `EnableConsoleScrlling == false` using `[3J` clears the back buffer. 

![hsuQ7XO 1](https://github.com/gui-cs/Terminal.Gui/assets/585482/127e11a9-5db6-456e-8b42-4735858495bc)

This is bad. 

`[0J` does not clear the back buffer, preserving what was typed above.

This is good.

![YoIwf9Y 1](https://github.com/gui-cs/Terminal.Gui/assets/585482/6cd5d588-0804-41f4-8649-0d4c0272da24)

Reply:
With `[0J`, if I set `_enableConsoleScrolling = true` in UI Catalog (in the source):

![ym4Auaj 1](https://github.com/gui-cs/Terminal.Gui/assets/585482/196161e6-5fe9-4f52-aca8-8d5948b3bc31)

Reply:
But you already read what you said about the WT not handle the buffer size of the running app, but only scroll back to the WT internal buffer. You have to use another terminal to see the difference.

Reply:
On `Linux` I also couldn't discover how to handle with the buffer size, but the terminal use scroll bar with backbuffer, so I think there is some way do control it. Unfortunately I already search so deep in the internet but I didn't found anything about it.